experimental/devices/pca9685/doc.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// Package pca9685 includes utilities to controls pca9685 module and servo motors.
+// More details
+//
+// See https://periph.io/device/pca9685/ for more details about the device.
+//
+// Based on https://github.com/wintersandroid/Android-Things-PCA9685
+//
+// Datasheets:
```

Comment:
For godoc to process this correctly, you need to keep an empty line between and no semi colon, e.g.:

```
// Datasheet
//
// https://www.nxp.com/docs/en/data-sheet/PCA9685.pdf
```

Use the original datasheet, not the copy from Adafruit.